src/endpoints/transcriptomics/index.ts

```diff
@@ -78,17 +80,31 @@ export const fetchSampleGeneExp = async (ensembl_gene_id: string): Promise<Sampl
                     ensembl_gene_id,
                 },
             },
-            _source: ['sample_id', 'x', 'y'],
+            _source: ['sample_id', 'x', 'y', 'age_at_biospecimen_collection_years'],
         },
     });
 
-    const points: SampleGeneExpPoint[] = body.hits.hits.map(hit => hit._source);
+    const points: SampleGeneExpPoint[] = body.hits.hits.map(hit => ({
```

Comment:
Is there a way to use map only once without affecting readability?

Reply:
Je peux faire un for qui fait tout en même temps mais c'est sur que ça va affecter la lisibilité

Reply:
J'ai mis des time pour voir et ça permet de passer de 2.5ms à 0.5ms donc j'ai fais le changement